oak_functions/proto/server.proto

```diff
@@ -16,17 +16,23 @@
 
 syntax = "proto3";
 
-package oak.remote_attestation;
+package oak.functions.server;
 
 option java_multiple_files = true;
-option java_package = "oak.remote_attestation";
+option java_package = "oak.functions.server";
+
+// Wrapper around the encrypted payload.
+message SecureRequest {
+  // Payload encrypted using the session key.
+  bytes encrypted_payload = 1;
```

Comment:
@ipetr0v FYI in your PR I meant basically adding the separate IV as a bytes field to this message, instead of concatenating everything in a single byte field.

Reply:
Got it
Will update once as part of synchronization with the handshaker